### PR TITLE
changelog for v0.5.3, add missing MRU utils

### PR DESCRIPTION
**Pull request by eshaan7**

### CODE REVIEW

File: docs/pages/build/framework/action/serialized-action.mdx [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 A serialized action is an action that encapsulates the execution and confirmation status along with logs and errors encountered during the execution of the acton.
 
+## SerializedAction
```
Comment (aashutoshrathi) [inline docs/pages/build/framework/action/serialized-action.mdx]:
```suggestion
## Serialized Action
```

File: docs/pages/build/framework/config.mdx [hunk 2]
```diff
@@ -152,3 +152,33 @@ with more support of data availability layers coming soon. No additional configu
...
+  vulcanRPC: string;
+  L1RPC: string;
+  isSandbox: boolean;
+  domain: Domain;
+  registryContract: string;
+  preferredDA: DA;
+  accounts: PublicAccount[];
+  logLevel: LogLevel;
+};
+```
```
Comment (aashutoshrathi) [inline docs/pages/build/framework/config.mdx]: good to have newline at the end

File: docs/pages/build/changelog.mdx [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Changelog [What's new]
 
+## v0.5.3
+
+#### Features / Improvements
+
+- Change datatype of `msgSender` and `signature` fields to `string` everywhere (such as `STF` args and `ActionParams` interface).
+Earlier, these were `AddressLike` / `SignatureLike` which weren't really useful without wrapping them in `String(v)` anyway.
```
Comment (kautukkundan) [inline docs/pages/build/changelog.mdx]: This line (Earlier...) is not needed here on the public docs

File: docs/pages/build/framework/action/serialized-action.mdx [hunk 4]
```diff
@@ -44,3 +45,23 @@ Returns the execution logs for the action. If not null, the execution succeeded
...
+```ts
+interface SerializedBlock {
+  height: number;
+  hash: string;
+  timestamp: number;
+  status: BlockStatus;
+  batchInfo?: BatchInfo;
+}
+
+interface SerializedActionAndBlock extends SerializedAction {
```
Comment (kautukkundan) [inline docs/pages/build/framework/action/serialized-action.mdx:62]: awkward naming convention `SerializedActionWithBlock` is better than `SerializedActionAndBlock`
can take this up later in sdk, ok for now

File: docs/pages/build/framework/action/serialized-action.mdx [hunk 5]
```diff
@@ -44,3 +45,23 @@ Returns the execution logs for the action. If not null, the execution succeeded
 Returns any errors that occurred during the execution of the action. It is null if the action has not yet been executed. Here the action is of type ``SerializedAction``.
 
 **Type:** `ExecutionError[] | null;`
+
+## Serialized Block
+
+In some places, the `SerializedAction` also includes block details. Then its datatype is `SerializedActionAndBlock` as given below
```
Comment (kautukkundan) [inline docs/pages/build/framework/action/serialized-action.mdx]: ambiguous statement - "in some places" 
what places? how do i get to those?

File: docs/pages/build/framework/config.mdx [hunk 6]
```diff
@@ -152,3 +152,33 @@ with more support of data availability layers coming soon. No additional configu
 10. `logLevel`: The log level of the micro-rollup. Defaults to `log`.
 
 This is used for viewing the logs of the micro-rollup during runtime. Logs are a useful tool for debugging and monitoring the micro-rollup and checking for any errors.
+
+
+## Accessing config values
+
+It's recommended to not use the `stackrConfig` object defined above directly in your rollup code, other than for initializing `MicroRollup`.
```
Comment (kautukkundan) [inline docs/pages/build/framework/config.mdx]:
```diff
+ It's not recommended
- It's recommended to not
```

**Review by kautukkundan — APPROVED** [review]
Looks good with a few changes

**Pull request merged**